onnxruntime/contrib_ops/rocm/bert/attention_impl.cu

```diff
@@ -496,6 +327,164 @@ Status LaunchDecoderAttentionKernel(
   }
 }
 
+
+constexpr int kPastSequenceLengthInputIndex = 6;
```

Comment:
should we still keep this part below in attention.cc?

Reply:
but will it be any issue to rename it back to .cc? Just want to be consistent with cuda code structure for people to easily compare and search for it.

Reply:
Yes, next pr will introduce many cuh header, which is caused by ck itself. Just rename it ahead to avoid future move.